.env.local.example

```
@@ -0,0 +1,34 @@
+# Sentry Error Tracking
+NEXT_PUBLIC_SENTRY_DSN=
+SENTRY_DSN=
+SENTRY_AUTH_TOKEN=
+
+# PostHog Product Analytics
+# Uses /ingest reverse proxy (see next.config.ts rewrites)
+NEXT_PUBLIC_POSTHOG_KEY=
+
+# Helicone LLM Observability (optional)
+# When set, OpenAI calls route through Helicone gateway
+HELICONE_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)